src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaUtils.java

```diff
@@ -106,6 +111,23 @@ public Project getProjectFromIndexPattern(String index) {
         }
         return new Project(index, null);
     }
+
+    public Project getProjectFromIndex(String index) {
+        
+        if (StringUtils.isEmpty(index)) {
+            return Project.EMPTY;
+        }
+        
+        Matcher matcher = reProjectFromIndex.matcher(index);
+        if (matcher.matches()) {
+            String name = matcher.group("name");
+            if("operations".equals(name) || "orphaned".equals(name)) {
```

Comment:
do we need to check for the preceeding '.' ?

Reply:
I don't believe so.  I believe elsewhere we represent them as 'projects' without the period and have special handling to add or remove it as needed